frame/support/src/storage/generator/double_map.rs

```diff
@@ -153,6 +153,14 @@ impl<K1, K2, V, G> storage::StorageDoubleMap<K1, K2, V> for G where
 		G::from_optional_value_to_query(unhashed::get(&Self::storage_double_map_final_key(k1, k2)))
 	}
 
+	fn try_get<KArg1, KArg2>(k1: KArg1, k2: KArg2) -> Result<Self::Query, ()>
```

Comment:
note that the `try_get` on StorageValue returns on success the type `T` instead of `Query`.

If your storage is `double_map ... => Option<u32>`, having `try_get` returning `Option<u32>` while it is always `Some(_)` isn't very useful.

(Similarly `mutate_exist` is using inner type `u32` directly)

But at the same time user should rather using `get` directly if its type is `Option<_>`. So all in all I think it is same.

Reply:
Yeah sounds plausible to me. I will change it in the way you proposed and use `V`